src/esp/gfx/PbrShader.cpp

```diff
@@ -113,14 +126,42 @@ PbrShader::PbrShader(Flags originalFlags, unsigned int lightCount)
                      : "")
       .addSource(flags_ & Flag::NormalTexture ? "#define NORMAL_TEXTURE\n" : "")
       .addSource(flags_ & Flag::ObjectId ? "#define OBJECT_ID\n" : "")
+
+      // Clearcoat layer
       .addSource(flags_ & Flag::ClearCoatLayer ? "#define CLEAR_COAT\n" : "")
+      .addSource(flags_ >= Flag::ClearCoatTexture
+                     ? "#define CLEAR_COAT_TEXTURE\n"
+                     : "")
+      .addSource(flags_ >= Flag::ClearCoatRoughnessTexture
+                     ? "#define CLEAR_COAT_ROUGHNESS_TEXTURE\n"
+                     : "")
+      .addSource(flags_ >= Flag::ClearCoatNormalTexture
+                     ? "#define CLEAR_COAT_NORMAL_TEXTURE\n"
+                     : "")
+
+      // Specular Layer
+      .addSource(flags_ & Flag::SpecularLayer ? "#define SPECULAR_LAYER\n" : "")
+      .addSource(flags_ >= Flag::SpecularLayerTexture
+                     ? "#define SPECULAR_LAYER_TEXTURE\n"
+                     : "")
+      .addSource(flags_ >= Flag::SpecularLayerColorTexture
+                     ? "#define SPECULAR_LAYER_COLOR_TEXTURE\n"
+                     : "")
+
+      // Anisotropy Layer
+      .addSource(flags_ & Flag::AnisotropyLayer ? "#define ANISOTROPY_LAYER\n"
+                                                : "")
+      .addSource(flags_ >= Flag::AnisotropyLayerTexture
+                     ? "#define ANISOTROPY_LAYER_TEXTURE\n"
+                     : "")
+
       .addSource(flags_ & Flag::PrecomputedTangent
                      ? "#define PRECOMPUTED_TANGENT\n"
                      : "")
       .addSource(flags_ & Flag::ImageBasedLighting
-                     ? "#define IMAGE_BASED_LIGHTING\n"
+                     ? "#define IMAGE_BASED_LIGHTING\n#define TONE_MAP\n"
```

Comment:
```suggestion
                     ? "#define IMAGE_BASED_LIGHTING\n"
                       "#define TONE_MAP\n"
```

feels more readable to me. Question is whether clang-format has the same opinion tho.

Reply:
I'll find out :D